src/panels/lovelace/components/hui-yaml-editor.ts

```diff
@@ -76,7 +98,12 @@ export class HuiYamlEditor extends HTMLElement {
             cm.replaceSelection(spaces);
           },
         },
+        gutters:
+          this._hass && computeRTL(this._hass)
+            ? ["rtl-gutter", "CodeMirror-linenumbers"]
+            : [],
       });
+      this.setScrollBarDirection(this._hass);
```

Comment:
No need for this, as `hass` has not changed

Reply:
It is needed. It doesn't work without this since inside when set hass is called - code mirror is not created yet.